src/cli.rs

```diff
@@ -0,0 +1,37 @@
+use std::convert::TryInto;
+
+use gettextrs::gettext;
+use gtk::prelude::{ApplicationExt, IsA};
+use gtk::{gio, glib};
+
+pub fn setup<A: IsA<gio::Application>>(app: A) -> A {
```

Comment:
I guess this can be part of main.rs? Or maybe you plan on adding more things to it and you want to keep it separate?

Reply:
No, I think there's nothing much to add.